tutorials/preprocessing/90_eyetracking_data.py

```diff
@@ -101,123 +108,107 @@
+# onset of the flash. The photodiode signal was recorded as a digital input channel in
+# the EEG and eye-tracking data. MNE loads these data as ``'stim'`` channels.
 #
-# In the example data, the DIN channel contains the onset of light flashes on
-# the screen. We now extract these events to visualize the pupil response. We will use
-# these later in this tutorial.
-
-events = mne.find_events(raw, shortest_event=1, min_duration=0.02, uint_cast=True)
-event_dict = {"flash": 3}
+# We'll extract the flash event onsets from both the EEG and eye-tracking data, as they
+# are necessary for aligning the EEG and eye-tracking data.
 
+et_events = mne.find_events(raw_et, min_duration=0.01, shortest_event=1, uint_cast=True)
+event_dict = {"Flash": 2}
+eeg_events = mne.find_events(raw_eeg, stim_channel="DIN3")
 
 # %%
-# Plot raw data
-# -------------
+# Plot the raw eye-tracking data
+# ------------------------------
 #
-# As the following plot shows, we now have a raw object with the eye tracker
-# data, eyeblink annotations and stimulus events (from the DIN channel).
-#
-# The plot also shows us that there is some noise in the data (not always
-# categorized as blinks). Also, notice that we have passed a custom `dict` into
-# the scalings argument of ``raw.plot``. This is necessary to make the eyegaze
-# channel traces legible when plotting, since the file contains pixel position
-# data (as opposed to eye angles, which are reported in radians). We also could
-# have simply passed ``scalings='auto'``.
-
-raw.plot(
-    events=events,
-    event_id={"Flash": 3},
+# Let's plot the raw eye-tracking data. We'll pass a custom `dict` into
+# the scalings argument to make the eyegaze channel traces legible when plotting,
+# since this file contains pixel position data (as opposed to eye angles,
+# which are reported in radians). We also could have simply passed ``scalings='auto'``.
+
+raw_et.plot(
+    events=et_events,
+    event_id=event_dict,
     event_color="g",
-    start=25,
-    duration=45,
+    duration=15,
     scalings=dict(eyegaze=1e3),
 )
 
 # %%
-# handling blink artifacts
+# Handling blink artifacts
 # ------------------------
-# We also notice that, naturally, there are blinks in our data, and these blink periods
+#
+# Naturally, there are blinks in our data, and these blink periods
 # occur within ``"BAD_blink"``  annotations. During blink periods, ``"eyegaze"``
 # coordinates are not reported, and ``"pupil"`` size data are ``0``. We don't want these
-# blink artifacts biasing our analysis, so we have two options: We can either drop the
-# blink periods from our data during epoching, or we can interpolate the missing data
-# during the blink periods. For this tutorial, let's interpolate the missing samples:
+# blink artifacts biasing our analysis, so we have two options: Drop the blink periods
+# from our data during epoching, or interpolate the missing data during the blink
+# periods. For this tutorial, let's interpolate the blink samples:
```

Comment:
Do we want to add a blurb here that this by default only affects the pupil responses?

Reply:
Good idea!